### ISSUE TITLE
Input rotated 90deg with e-ink device

### ISSUE THREAD

**Issue by markwmuller** [dkrivoruchko/ScreenStream#98]
I'm using latest version from f-droid (3.4.9). When using screenstream with an Onyx Boox Note 2 e-ink device, I'm having weird behaviour where the input appears rotated 90 degrees. A video of this is shown here:  https://youtu.be/ks45WFmvws0 

The e-ink tablet does something weird with the android display when using the pen, where the actual display is only updated at some delay even if the pen appears immediately on the device, but the strange 90degree rotation is unexpected. This may be unique to this device, and how it needs to hack the android system to make it useable with E-ink, but it would be fantastic if I could use this tool without issue. 

Note that, if I use the tablet in landscape, the issue is gone (but other problems unrelated to screenshare make this less desirable). 

In short, the behaviour is odd; it appears that screenstream intercepts the inputs and then communicates them onwards to the underlying system, potentially violating some internal rotation thereby causing the input to appear twice on the tablet, once at a 90degree offset to where it was intended.

I guess this is likely not a bug of screenstream, but maybe there's an easy workaround for such devices that would allow me working in portrait mode?

**Comment by dkrivoruchko** [comment]
If you turn off ScreenStream will this app where you draw behave the same strange way?
ScreenStream has an option to rotate screen. Try to test with it on

**Comment by markwmuller** [comment]
> If you turn off ScreenStream will this app where you draw behave the same strange way?
No -- as long as I use only the device, then the devices is fine.

>  ScreenStream has an option to rotate screen. Try to test with it on
This works, if I rotate the device to be landscape, and then use this to project at a 90deg view. This allows to display as portrait on my PC, but is a little clunky as things are now 90deg rotated (e.g. text in menus). 

Thanks for ideas!
